A3A/addons/core/functions/CREATE/fn_AIVEHinit.sqf

```diff
@@ -204,6 +204,19 @@ if (_side != teamPlayer) then
 	}];
 };
 
+if(_veh isKindOf "Air") then
+{
+    //Start airspace control script if rebel player enters
+    _veh addEventHandler ["GetIn", {
+		params ["_veh", "_role", "_unit"];
+		if((side (group _unit) == Invaders) || (side (group _unit) == Occupants) && {isPlayer _unit}) then
+		{
+			// TODO: check this isn't spammed
+			[_veh] spawn A3A_fnc_airspaceControl;
+		};
+    }];
+};
+
```

Comment:
What's the purpose of this?

Reply:
was messing around with airspace control, thought, why ai doesn't react on ai? better comment it out(not delete) until something comes out of this https://github.com/official-antistasi-community/A3-Antistasi/issues/3246, ... or test it and see what it does/break

Reply:
> was messing around with airspace control, thought, why ai doesn't react on ai? better comment it out(not delete) until something comes out of this [official-antistasi-community#3246](https://github.com/official-antistasi-community/A3-Antistasi/issues/3246), ... or test it and see what it does/break

https://github.com/official-antistasi-community/A3-Antistasi/pull/3250

Reply:
nuh, I'll comment it out and leave it as is until, I'll start porting some changes from C.V.